docs/proposals/Gallery/README.md

```diff
@@ -0,0 +1,58 @@
+# Component Name
+
+The `Gallery` component will accept a list of attachments for images and display
```

Comment:
We mostly refer to images in this proposal, but it isn't clear if we also mean other files. We should be clear. (But, probably through some discussion first!)